app-misc/ola/ola-0.10.7.ebuild

```diff
@@ -0,0 +1,100 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python2_7 python3_{6,7} )
```

Comment:
python2 is being removed from ::gentoo, don't add any new packages depending on it.

Reply:
Done